advanced/Scripts/piholeDebug.sh

```diff
@@ -336,7 +342,11 @@ countdown() {
   printf "::: Logging will automatically teminate in ${TIMEOUT} seconds\n"
   while [ $tuvix -ge 1 ]
   do
-    printf ":::\t${tuvix} seconds left. \r"
+    if ! ${automated} ; then
+      printf ":::\t${tuvix} seconds left. \r"
```

Comment:
There shouldn't be a new line, it has a carriage return to overwrite the counter on a terminal display.